{"name": "test_01[iozone -r 1M -i 0 -i 1  -s 20G -I]", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <Case.test_fio.TestIOzone object at 0x0000029FC8BD1A90>\ncmd = 'iozone -r 1M -i 0 -i 1  -s 20G -I'\n\n    @pytest.mark.parametrize('cmd', [i[0] for i in data])\n    def test_01(self, cmd):\n        ioz_res = fp.send(cmd)\n>       fp.assert_Iozone(ioz_res)\n\ntest_fio.py:16: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Utils.Base_init.BspTest object at 0x0000029FC8AD7C50>\nioz_res = '\\tIozone: Performance Test of File I/O\\n\\t        Version $Revision: 3.489 $\\n\\t\\tCompiled for 64 bit mode.\\n\\t\\tBuil...  2049925                                                                                  \\n\\niozone test complete.\\n'\n\n    def assert_Iozone(self, ioz_res):\n        op = OperationData('iozone.csv')\n        dict_list = dict(ChainMap(*op.get_data_dict()))\n        re_result = re.findall('[0-9]{8}\\s+[0-9]{4}\\s+[0-9]{7}\\s+[0-9]{7}\\s+[0-9]{7}\\s+[0-9]{7}', ioz_res)\n>       re_list = re_result[0].split('  ')\nE       IndexError: list index out of range\n\n..\\Utils\\Base_init.py:52: IndexError"}, "attachments": [{"name": "log", "source": "13198631-0b1a-4cff-8720-13d3046c98a5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "cmd", "value": "'iozone -r 1M -i 0 -i 1  -s 20G -I'"}], "start": 1740379563878, "stop": 1740379644278, "uuid": "4b29d0ad-7012-442d-8a6b-67cc4b528165", "historyId": "1800ecaa154f528cbb1cc87e4fcc3d7a", "testCaseId": "01a55d7010f48ebe0c3cf2503891f546", "fullName": "Case.test_fio.TestIOzone#test_01", "labels": [{"name": "parentSuite", "value": "Case"}, {"name": "suite", "value": "test_fio"}, {"name": "subSuite", "value": "TestIOzone"}, {"name": "host", "value": "Meiyi-EXT"}, {"name": "thread", "value": "29412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Case.test_fio"}]}